rt_dashboard/templates/rt_dashboard/dashboard.html

```diff
@@ -103,7 +103,14 @@ <h1>Tasks on <strong{% if queue.name == '[failed]' %} class="failed"{% endif %}>
                 <i class="icon-trash icon-white"></i> Delete
             </a>
         {% endif %}
-        This list below contains all the registered tasks on queue <strong>{{ queue.name }}</strong>, sorted by age (oldest on top).</p>
+        This list below contains all the registered tasks on queue
+        <strong>{{ queue.name }}</strong>, sorted by age
+        {% if queue.name.startswith('[') %}
+            (newest on top).
```

Comment:
This won't be necessarily right for the `[running]` queue, right?

Reply:
That is correct, but I chose to ignore that for now. It was also wrong before.